mpt_extension_sdk/mpt_http/mpt.py

```diff
@@ -207,7 +207,7 @@ def get_agreements_by_next_sync(mpt_client, next_sync_parameter):
 
 
 def get_agreements_by_3yc_enroll_status(
-    mpt_client: MPTClient, enroll_statuses: tuple[str], status: str = "Active"
+    mpt_client: MPTClient, enroll_statuses: Enum, status: str = "Active"
```

Comment:
But why do we anything related to the Adobe's 3yc in SDK???

Reply:
You are right, we do not need this here.